systemtest/produce_consume_basic_test.go

```diff
@@ -189,11 +192,13 @@ func TestProduceConsumeMultipleTopics(t *testing.T) {
 }
 
 type produceConsumeCfg struct {
-	events   int
-	producer apmqueue.Producer
-	consumer apmqueue.Consumer
-	records  *atomic.Int64
-	timeout  time.Duration
+	events          int
+	replay          int
+	expectedRecords int
```

Comment:
How about naming this `expectedRecordsCount` (and maybe making the same change to the other integers)? Based on the name, I expected a slice of records, not a count.

Reply:
Done!